packages/eas-cli/src/build/ios/version.ts

```diff
@@ -190,5 +219,12 @@ function ensureStaticConfigExists(projectDir: string): void {
 }
 
 export function evaluateTemplateString(s: string, vars: Record<string, any>): string {
-  return s.replace(/\$\((\w+)\)/g, (match, key) => (vars.hasOwnProperty(key) ? vars[key] : match));
+  return s.replace(/\$\((\w+)\)/g, (match, key) => {
+    if (vars.hasOwnProperty(key)) {
+      const value = String(vars[key]);
+      return value.startsWith('"') ? value.slice(1, -1) : value;
```

Comment:
Maybe it's worth writing a util for this, I think we have this in few places now.